tests/EvmAcceptanceTests/test/otterscan/ots_getTransactionBySenderAndNonce.ts

```diff
@@ -24,7 +24,7 @@ describe(`Otterscan api tests: ${METHOD}`, function () {
     const accounts = Array.from({length: ACCOUNTS_COUNT}, (v, k) =>
       ethers.Wallet.createRandom().connect(ethers.provider)
     );
-
+    
```

Comment:
Please run `npm run lint`

Reply:
I'm sure you believe that keeping the project clean is part of delivering it in the long term. `npm run lint` as you know doesn't prevent you from going forward. It fixes the code style problems automatically in a few seconds.

Reply:
And yet, it's a week later and we're still not merged :-(